dub.json

```diff
@@ -21,17 +25,33 @@
         },
     },
 
+    "subPackages": [
+        {
+            "name": "configure",
+            "targetType": "executable",
```

Comment:
Is there any practical reason to also expose configure script as sub-package? That feels a bit semantically wrong to me as configure script is used as part of build process of other configurations, it is hardly a thing on its own.